lib/ui/profile/interactor/profile_data.dart

```diff
@@ -44,13 +44,24 @@ class ProfileData {
     return ProfileData(
       name: name,
       account: account,
-      avatarUrl: null,
+      avatarUrl: avatar,
       bio: bio,
       s3Identity: s3Identity,
       avatar: avatar,
     );
   }
 
+  ProfileData updateBio(String? bio) => ProfileData(
+        name: name,
+        account: account,
+        bio: bio,
+        avatar: avatar,
+        avatarUrl: avatarUrl,
```

Comment:
@n13  do we need avatar and avatarUrl ?

Reply:
Yes

Reply:
avatarURL is a resolved URL

avatar is the image name, like xyz.jpg

Reply:
and what is the avatar used for?